data/tekton-pipelines/okd/windows11-installer-example-pipelinerun.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
Other example pipelineruns are placed here: https://github.com/kubevirt/kubevirt-tekton-tasks/tree/main/examples/pipelines

Reply:
I will do a follow up PR after this one is merged